src/test/java/org/apache/geronimo/microprofile/impl/jwtauth/jwt/JwksServer.java

```diff
@@ -0,0 +1,56 @@
+package org.apache.geronimo.microprofile.impl.jwtauth.jwt;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+class JwksServer {
```

Comment:
why not `HttpServer`? - not a blocker more curious

Reply:
I guess, it was, because we just had the code lying around.